int_experian/int_experian_sgjc/cartridge/static/default/js/edqUtils.js

```diff
@@ -384,15 +394,18 @@ function edqSetPhoneValidationConfiguration() {
  */
 function edqSetGlobalIntuitiveConfiguration() {
 	window.EdqConfig.GLOBAL_INTUITIVE_AUTH_TOKEN=edqAuthorizationToken;
-	if(edqCountryLineSelector != null)
+	if(edqCountryLineSelector != null) {
 		window.EdqConfig.GLOBAL_INTUITIVE_ISO3_COUNTRY=countryAlpha3(edqCountryLineSelector.value);
-	else
+	}
+	else {
 		window.EdqConfig.GLOBAL_INTUITIVE_ISO3_COUNTRY=countryAlpha3(vDefaultCountry);
+	}
 	/**
 	* Feature 118583
 	* Configuration option to include Data Sets for Global Intuitive*/
-	if (edqDataSetUsage)
-		window.EdqConfig.GLOBAL_INTUITIVE_DATASET=edqDataSetCode;
+	if (edqDataSetUsage) {
+		window.EdqConfig.GLOBAL_INTUITIVE_DATASET=edqDataSetCode; 
```

Comment:
Let's put a space between the equals:

```javascript
window.EdqConfig.GLOBAL_INTUITIVE_DATASET = edqDataSetCode; 
```

Once we have a build going hopefully we can detect these kinds of things automatically.